### ISSUE TITLE
Deprecate the use of number ranges for abundance

### ISSUE THREAD

**Issue by kitenetter** [NERC-CEH/irecord-app#295]
There have been a number of conversations about the pros and cons of using number ranges in the app, but it seems to me that most schemes are moving away from using ranges. In particular, BC have asked us to disallow number ranges for butterflies and moths.

I would suggest that we disallow ranges for all taxon groups, unless they have been specifically requested. I know that British Dragonfly Society use ranges in their custom abundance attributes, and I don't think these should be changed.

@kazlauskis are you aware of any other schemes that do wish to use number ranges?

**Comment by sacrevert** [comment]
@japonicus Can you comment from BSBI perspective. I note that the iRecord app currently displays the following for a plant recorded as a casual record
![Screenshot (4 Dec 2024 12_07_38)](https://github.com/user-attachments/assets/46456c26-1623-4a96-9beb-5ea78e3cb3ee)
but, through the Plant Survey (square-bashing), it currently allows for "DAFOR, LA, LF or count", i.e. ranges are already disallowed.

**Comment by japonicus** [comment]
@sacrevert I don't have strong opinions about this. In general, I'd generally prefer that recorders gave a precise count rather than a range, though for a large population a range may be reasonable. For the sake of simplicity in the iRecord app I think I'd be happy for range options to be dropped. We do currently allow range abundances in BSBI's own app, but the default option is 'count'.

Separately, in our own app we're planning to rethink how DAFOR abundances are applied, because they are mostly misused and make no sense if the survey area over which the assessment was made is unspecified. e.g. for a monad survey it's not necessarily clear whether a recorder was evaluating the whole 1000m square when stating that something is Frequent. We'll probably add an extra field to specify the area unit.

In an app such as iRecord, that is partly intended for a non-specialist audience, the DAFOR scale terms are often likely to be misunderstood and not applied in their technical sense. There could be a case for keeping them low profile with a simple count field as the prominent option for abundance.

@kwal2 may have views about this

**Comment by sacrevert** [comment]
I think for the moment we can bring the two Abundance attribute presentations for vascular plants in line with each other (i.e. giving the DAFOR/LF/LA/precise count field as in currently the case within the Plant Survey). Precision notwithstanding, I think that the DAFOR option is in widespread enough use to be left in; it's vagueness is often part of its attraction.

**Comment by japonicus** [comment]
Part of the problem with DAFOR is that it is widely used but that there isn't a consistent sense of what it means, so that any classifications applied are not useful for any sort of analysis.

I'd like to try to tighten things. That might (perhaps probably will) fail, but I'm not happy with the status quo where DAFOR etc. are so ill defined and over applied, while recorders think that they are doing something useful.

**Comment by sacrevert** [comment]
I agree, but not everything offered has to be amenable to aggregated analysis. I think we should continue to offer it simply because some individuals find it useful.

**Comment by sacrevert** [comment]
Also number item 7 in list at #211 , where entering a zero count in the plant survey abundance field does not result in a null record (the opposite behaviour to the plant survey on the website)
